mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -203,6 +203,7 @@ class GBTClassifier @Since("1.4.0") (
     } else {
       GradientBoostedTrees.run(trainDataset, boostingStrategy, $(seed), $(featureSubsetStrategy))
     }
+    baseLearners.map(tree => copyValues(tree))
```

Comment:
maybe `baseLearners.foreach(copyValues)` ?

Reply:
Thanks! I will change to ```baseLearners.foreach(copyValues(_))``` because ```copyValues``` takes two arguments.